website/docs/help-and-support/troubleshooting-guide/deployment-errors.md

```diff
@@ -105,3 +105,14 @@ sudo docker-compose up -d --force-recreate appsmith-internal-server
 ```
 
 In a minute or two, the server should now come up and be ready.
+
+
+## Email errors
+
+If you are using Docker and are having trouble with invitation emails not being sent, even though the Admin email setup is able to send test emails, it may be due to an issue with the configuration of the Docker container.
+
+If you aren't receiving the invitation email, please check the value for ```APPSMITH_REPLY_TO``` in your **docker.env** file. If this value is empty, please set it to the same email address you are using for ```APPSMITH_MAIL_FROM``` and **restart the application**.
```

Comment:
If the above comment is true then instead of docker.env let's say "in the configuration file"